### PR TITLE
Use Kubernetes 1.16.2

### PR DESCRIPTION
**Pull request by dmage**

### CODE REVIEW

File: tools.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// +build tools
+
+package tools
```
[the remark this answers is not in the document]
Comment (dmage) [reply tools.go]: Why do we invent our own convention? https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module

[the remark this answers is not in the document]
Comment (dmage) [reply tools.go]: https://github.com/search?q=org%3Aopenshift+filename%3Atools.go+%22%2Bbuild+tools%22

And I'd prefer to move this file to hack/codegen and keep script dependencies near the scripts.

[the remark this answers is not in the document]
Comment (dmage) [reply tools.go]: I added a comment to align with other repos

[the remark this answers is not in the document]
Comment (adambkaplan) [reply tools.go]: @dmage I didn't realize there was an existing convention. The apiserver team seems to have adopted a different convention. Now that I am aware of the `tools` convention, let's keep it.

https://github.com/search?q=org%3Aopenshift+filename%3Adoc.go+%22%2Bbuild+tools%22

**Pull request merged**